wandb/integration/keras/keras.py

```diff
@@ -1006,19 +1025,46 @@ def _save_model(self, epoch):
             self.save_model = False
             self.save_model_as_artifact = True
 
-    def _save_model_as_artifact(self, epoch):
+    def _save_model_as_artifact(self, epoch, logs):
         if wandb.run.disabled:
             return
 
-        # Save the model in the SavedModel format.
-        # TODO: Replace this manual artifact creation with the `log_model` method
-        # after `log_model` is released from beta.
+        try:
+            if self.save_best_only:
+                current = logs.get(self.monitor)
+                if current is None:
+                    wandb.termwarn(
+                        f"Can save best model only with {self.monitor}"
+                        "available, skipping."
+                    )
```

Comment:
Could you reword this one? I'm not clear what is the warning suppose to mean

Reply:
sure.